model/task/results.go

```diff
@@ -133,3 +134,62 @@ func FilterTasksOnStatus(tasks []Task, statuses ...string) []Task {
 
 	return out
 }
+
+type Stat struct {
+	Name  string `json:"name"`
+	Count int    `json:"count"`
+}
+type ResultCountList struct {
+	Total              []Stat `json:"total"`
+	Inactive           []Stat `json:"inactive"`
+	Unstarted          []Stat `json:"unstarted"`
+	Started            []Stat `json:"started"`
+	Succeeded          []Stat `json:"success"`
+	Failed             []Stat `json:"failed"`
+	SetupFailed        []Stat `json:"setup-failed"`
+	SystemFailed       []Stat `json:"system-failed"`
+	SystemUnresponsive []Stat `json:"system-unresponsive"`
+	SystemTimedOut     []Stat `json:"system-timed-out"`
+	TestTimedOut       []Stat `json:"test-timed-out"`
+}
+
+func GetResultCountList(statuses []StatusItem) ResultCountList {
```

Comment:
can't this be pushed back up to the database in the aggregation?

Reply:
I pushed the reshaping into lists back into the aggregation, but still get the totals on our side.
I don't see how to calculate the totals on the db side in the same aggregation.